www/Gemfile

```diff
@@ -18,6 +18,7 @@ gem 'middleman-compass', '>= 4.0.0'
 gem 'middleman-livereload'
 gem 'middleman-sprockets', '>= 4.0.0'
 gem 'middleman-syntax'
+gem 'middleman-search_engine_sitemap', '~> 1.4'
```

Comment:
Can we move this in order to be in alphabetical order?

This would solve:
```
www/Gemfile:21:1: C: Gems should be sorted in an alphabetical order within their section of the Gemfile. Gem middleman-search_engine_sitemap should appear before middleman-syntax.
```